apps/frontend/src/components/dashboard/links/link/link.tsx

```diff
@@ -23,8 +24,12 @@ export const LinkBlock = component$(({ id, urlKey, url, favicon, createdAt, expi
         <div class="grid grid-cols-12 gap-4">
           {/* First column with the link and favicon */}
           <div class="flex items-center space-x-3 col-span-6">
-            <div class="hidden sm:block">
-              <img src={favicon || `https://www.google.com/s2/favicons?sz=64&domain_url=${url}`} class="w-8 h-8 rounded-full" />
+            <div class="hidden sm:block flex-shrink-0">
+              <img
+                alt={getDomainNameFromUrl(url)}
```

Comment:
Consider use the native URL api

```suggestion
                alt={new URL(url).hostname}
```